src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -228,10 +279,10 @@ export class BitcoinAdapter extends BaseProtocolAdapter {
       redirect: 'follow' as RequestRedirect,
     };
 
-    const transactionId = await fetch(`${nodeUrl}/tx`, requestOptions)
+    const transactionIdPromise = await fetch(`${nodeUrl}/tx`, requestOptions)
```

Comment:
Why are you using `await` and `then` here?

Reply:
Fixed it.